lib/omniauth/strategies/openid_connect.rb

```diff
@@ -221,8 +221,8 @@ def access_token
         return @access_token if @access_token
 
         @access_token = client.access_token!(
-          scope: (options.scope if options.send_scope_to_token_endpoint),
-          client_auth_method: options.client_auth_method
+          client_auth_method: options.client_auth_method,
+          scope: (options.scope if options.send_scope_to_token_endpoint)
```

Comment:
From my quick test, the original code should be fine:

```ruby
[1] pry(main)> args = [{ scope: "foo", client_auth_method: "bar" }]
=> [{:scope=>"foo", :client_auth_method=>"bar"}]
[2] pry(main)> options = args.extract_options!
=> {:scope=>"foo", :client_auth_method=>"bar"}
[3] pry(main)> args.first
=> nil
```

It's passing a hash to `access_token!` and `extract_options!` removes that hash from args leaving args as an empty array.

Reply:
Ok, thanks. I'll close this for now then.